test/process-timeouts/index.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const assert = require('assert');
+const { resolve } = require('path');
+const { exec } = require('child_process');
+const libExecutable = resolve(__dirname, '../../dist/bin/cli.js');
+
+exec(`${libExecutable} --max-parallel 1 test/reusing-processes-2/index.1.spec.js test/reusing-processes-2/index.2.spec.js`, {
+    cwd: resolve(__dirname, '../../'),
+}, err => {
+    assert(err);
```

Comment:
This is a bit generic. It would be great to add a second argument saying what kind of error happened.

Reply:
It always exits with code 1 I think. It would tell us nothing more about the error.

Reply:
I mean that the error that you will get if this test fails looks like this now:

> AssertionError [ERR_ASSERTION]: null == true
>    at b (repl:1:11)

And what I think is that the test error should ideally be understandable even without debugging the test. So simple change like that would make things much easier to debug:

```
assert(err, 'CLI was supposed to exit with an error but it didn\'t')
```

Reply:
Ok, changed it

Reply:
Fixed